packages/screens/CairoPOC/CairoPOCScreen.tsx

```diff
@@ -0,0 +1,163 @@
+import { useAccount, useContract } from "@starknet-react/core";
+import { useCallback, useEffect, useState } from "react";
+import { View } from "react-native";
+import { type Abi, shortString } from "starknet";
+
+import { Todo } from "./types";
+
+import { BrandText } from "@/components/BrandText";
+import { ScreenContainer } from "@/components/ScreenContainer";
+import { TertiaryBox } from "@/components/boxes/TertiaryBox";
+import { PrimaryButton } from "@/components/buttons/PrimaryButton";
+import { useFeedbacks } from "@/context/FeedbacksProvider";
+import { useSelectedNetworkInfo } from "@/hooks/useSelectedNetwork";
+import { StarknetNetworkInfo } from "@/networks";
+import { abi as todoListAbi } from "@/starknet-contract-clients/todo_list/abi.json";
+
+export const CairoPOCScreen = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [currentTx, setCurrentTx] = useState<string | null>(null);
+
+  const { account } = useAccount();
+  const selectedNetwork = useSelectedNetworkInfo();
+  const { setToast } = useFeedbacks();
+
+  const { contract } = useContract({
+    // @ts-expect-error: TODO: make proper type
+    abi: todoListAbi satisfies Abi,
```

Comment:
Should we do that like this?
```suggestion
    // TODO: make proper type
    abi: todoListAbi as Abi,
```